docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+------------------------------------------------------------ 
+
+### Step 15: Compile new Neuron code
+
+
+PS: Useful commands:  
+
+```
+cd /c/Users/User/Neuron/nrn/build
+``` 
+to reach files outside of `C:\msys64\`.
+
+```
+Echo $PATH
+Which gcc 
+Which cmake 
+```
+To compile new neuron code, you can (always from a `mingw64` terminal): 
+
+#### Step 15.A: run win_build_cmake.sh 
+
+From nrn file where the source code is (example: `C:\Users\User\Neuron\nrn\ci` or `C:\msys64\home\User\Neuron\nrn\ci` if you copy a version here) run: 
+```
+.\win_build_cmake.sh
+```
+or 
+ 
+#### Step 15.B: Run "cmake .." 
+
+From nrn file where is the source code (example: C:\Users\User\Neuron\nrn or C:\msys64\home\User\Neuron\nrn if you copy a version here) run: 
+
+```
+.\cmake
+```
+
+or 
+
+#### Step 15.C: ninja
+
+Run `ninja` from build file (example: `C:\Users\User\Neuron\nrn\build` or `C:\msys64\home\User\Neuron\nrn\build` if you copy a version here) 
+```
+.\ninja
+``` 
+
+### Step 16: Dependences
+
+Python will not have all dependences, so you can install them doing:  
+
+```
+python -m pip install plotly 
+python -m pip install matplotlib 
+python -m pip install pandas 
+``` 
+
+
+## Creating a new environment for neuron development 
+It my be useful maintain the original version of neuron installed (at  for example) and have a new version that you can do your changes in code. We will set up this new environment and version.
+
+### Step 17: Clone neuron-git inside `msys64`
+We create a new folder inside the `msys64`:  `C:\msys64\home\User\Neuron\nrn`
+and clone neuron-git: 
+```
+git clone https://github.com/neuronsimulator/nrn
+```
+ 
+### Step 18: Set the configuration options: 
+
+Get inside `Neuron\nrn\build` file (example: `C:\Users\User\Neuron\nrn\build` or `C:\msys64\home\User\Neuron\nrn` if you copy a version here) and do:  
+```
+cmake .. -GNinja -DCMAKE_INSTALL_PREFIX=install -DNRN_ENABLE_RX3D=OFF -DNRN_ENABLE_PYTHON_DYNAMIC=ON -DNRN_ENABLE_MPI_DYNAMIC=ON 
+```
+
+Another configuration that can be useful is:
+```
+cmake .. -GNinja -DCMAKE_INSTALL_PREFIX=install -DNRN_ENABLE_RX3D=OFF -DNRN_ENABLE_PYTHON_DYNAMIC=ON -DNRN_ENABLE_MPI_DYNAMIC=ON -DPYTHON_EXECUTABLE=/c/Python312/python.exe -DCMAKE_BUILD_TYPE=FastDebug 
+```
+ 
+### Step 19: Compiling  
+A `Ninja install` and `ninja setup_exe` (instead `make setup_exe`) may be useful/ necessary for the first time. 
+
+The command 
+`ninja` may be enough for further changes from `C:\msys64\home\User\Neuron\nrn\build` folder. See Step 15.C. 
+ 
+Remembering useful files path: 
+```
+C:\Users\User\Neuron\nrn\build\src\mswin 
+C:\nrn\bin 
+C:\Users\User\Desktop\NEURON 9.0.0 AMD64 
+C:\Users\User\Neuron\nrn\build\bin 
+C:\msys64\home\User\Neuron\nrn\build\bin 
+ ```
+
+
+## Common problems:
+
+ 
+
+Do **Step 10** if you face something like:
+
+
+```bash
```

Comment:
powershell

Reply:
or maybe just no syntax formatting... but it isn't bash

Reply:
done